### DISCUSSION TITLE
What version of Wine does Whisky use?

### DISCUSSION THREAD

**Discussion by sofakng** [Whisky-App/Whisky#625] [General]
Does Whisky use a CrossOver version of Wine, Gcenx's versions of Wine, or something else maybe?

**Comment by IsaacMarovitz** [comment]
Whisky currently uses a modified version of CrossOver Wine 7.7.0 see:
- https://github.com/Whisky-App/WhiskyBuilder
- https://github.com/Whisky-App/homebrew-apple
